### DISCUSSION TITLE
Thonny won't import the BME280 module and gives SSL: CERTIFICATE_VERIFY_FAILED error

### DISCUSSION THREAD

**Discussion by Badflash2** [micropython/micropython#13670] [Core Development]
I suspect the Windows Update this morning is the cause as things were fine yesterday, now Thonny is acting odd. I'm using a Pi Pico W with the latest RPI_PICO_W-20240105-v1.22.1.uf2 file. I was trying to use a BME280, got the driver file and added it to the Pico. When I tried to import the driver it said it could not find it.

I went to Thonny Tools and attempted to use PyPI to import the file and got this error:
<urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1007)>

Try opening the package directly:
bme280

Opening directly just gave me more grief. 

Install from local file also refused to work.

Any help would be appreciated.

**Comment by bixb922** [comment]
PyPi modules seldom are Micropython compatible nor are they AFAIK installable to a microcontroller. See https://github.com/mcauser/awesome-micropython for a list of BME280 drivers (and for many other Micropython packages). These drivers are usually just one .py file. Copy that with Thonny or mpremote to the /lib folder of the microcontroller and you are good to go.

**Comment by Badflash2** [comment]
It was windows. Someone suggested I use a file called lets-encrypt-r3. It created a cert that let it work.
